### PR TITLE
Change to using .yaml instead of .yml

### PR DESCRIPTION
**Pull request by cdrage**
Use .yaml instead of .yml (.yml is considered "deprecated" since it was
once used when Windows only supported three-letter-extensions).

### CODE REVIEW

**Comment by mohammedzee1000** [comment]
This will require updating the index ci https://github.com/CentOS/container-pipeline-service/blob/master/testindex/Engine.py#L42

**Comment by mohammedzee1000** [comment]
https://github.com/CentOS/container-pipeline-service/pull/447

**Comment by roidelapluie** [comment]
What is the source of the initial statement?

**Comment by mohammedzee1000** [comment]
@roidelapluie didnt catch what you meant by that, sorry. Could you please elaborate?

**Comment by navidshaikh** [comment]
> What is the source of the initial statement?

@roidelapluie : [This](http://www.yaml.org/faq.html) FAQ on yaml.org should be a guideline for this change.

**Comment by mohammedzee1000** [comment]
I will get it done :)

**Comment by mohammedzee1000** [comment]
@cdrage ^ can you update your PR to rename instead of remove?

**Comment by mohammedzee1000** [comment]
@cdrage also note if you actually renamed the files in your branch and they are not appearing here, then it might be because of this https://github.com/CentOS/container-index/blob/master/.gitignore#L10

**Comment by cdrage** [comment]
@mohammedzee1000 @bamachrn Woops. That .gitignore ignored those yaml files. I've updated the PR.

**Comment by bamachrn** [comment]
@cdrage @mohammedzee1000 I believe we can take this in after https://github.com/CentOS/container-pipeline-service/pull/450 is in. @cdrage if you would like to take off the conflicts in the mean while.

**Comment by cdrage** [comment]
@bamachrn done!

**Comment by bamachrn** [comment]
@mohammedzee1000 @cdrage want to have a look what is going on here?

**Comment by mohammedzee1000** [comment]
@cdrage do you want to rebase this?

**Comment by cdrage** [comment]
@mohammedzee1000 done

**Comment by mohammedzee1000** [comment]
Well apparently we got conflicts again. @cdrage sorry can you rebase il get it merged as soon as that is done :)

**Comment by cdrage** [comment]
@mohammedzee1000 rebased again

File: index.d/fabric8-services.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Projects:
```
Comment (mohammedzee1000) [inline index.d/fabric8-services.yaml]: This entry has been removed. This introduces inconsistency with current state of index.

File: index.d/minishift.yml [hunk 2]
```diff
@@ -1,13 +0,0 @@
-Projects:
```
Comment (mohammedzee1000) [inline index.d/minishift.yml]: This is a valid entry and we dont want this removed. Again inconsistency.

**Pull request merged**